src/pytest_forked/__init__.py

```diff
@@ -71,13 +71,8 @@ def runforked():
 
 
 def report_process_crash(item, result):
-    try:
-        from _pytest.compat import getfslineno
-    except ImportError:
-        # pytest<4.2
-        path, lineno = item._getfslineno()
-    else:
-        path, lineno = getfslineno(item)
+    from _pytest._code.source import getfslineno
```

Comment:
It's been relocated to `.code` 27 days ago: https://github.com/pytest-dev/pytest/pull/7171/files#diff-ef66cddae564a1d8faabcab1bc4dccb4L282

But it's also accessible via `__init__` so just removing `.source` would "fix" it.

Reply:
its entirely possible this has been incorrect for years and was just missed

Reply:
No, before the mentioned relocation, it was correct.